chain/x/zoracle/alias.go

```diff
@@ -6,9 +6,10 @@ import (
 )
 
 const (
-	ModuleName = types.ModuleName
-	RouterKey  = types.RouterKey
-	StoreKey   = types.StoreKey
+	ModuleName        = types.ModuleName
+	DefaultParamspace = types.DefaultParamspace
```

Comment:
Why is this still using lowercase s? Let's do `DefaultParamSpace`

Reply:
ditto for all places. like 10 of them.

Reply:
Cosmos-SDK is very consistently inconsistent. They actually use `DefaultParamspace` in all places despite using `paramSpace` also in many places.